README.md

```diff
@@ -155,6 +157,19 @@ If the barrier method returns `false`, message will be returned into the queue f
 
 > **_NOTE:_**  See [SqsPermissionConsumer.ts](./packages/sns/test/consumers/SnsSqsPermissionConsumer.ts) for a practical example.
 
+### Dead Letter Queue
+A dead letter queue is a queue where messages that cannot be processed successfully are stored. It serves as a holding area for messages that have encountered errors or exceptions during processing, allowing developers to review and handle them later.
+To create a dead letter queue, you need to specify the `deadLetterQueue` parameter within options on the consumer configuration. The parameter should contain the following fields:
+- `creationConfig`: configuration for queue to create, if one does not exist. Should not be specified together with the `locatorConfig`
+- `locatorConfig`: configuration for resolving existing queue. Should not be specified together with the `creationConfig`
+- `redrivePolicy`: an object that contains the following fields:
+  - `maxReceiveCount`: the number of times a message can be received before being moved to the DLQ.
+
+> **_NOTE:_**  DLQ is not available for AMQP consumers.
+
+> **_NOTE:_**  DLQ could produce message duplication if used together with the barrier pattern (due to the 
+    retry mechanism), although this is unlikely to happen in practice. 
```

Comment:
I would clarify it - if a service crashes after receiving retryLater result from a barrier and creating a new message for the retry, but before the original one is consumed, there will be a duplicate, because those are not being done as an atomary operation.
It would also be helpful to clarify how the barrier works when used together with DLQ, that we do not simply return message to the queue, but create a new one instead, in order to avoid exhausting DLQ limits early.

Reply:
Makes sense, I will apply your suggestions to the node

Reply:
Done, please have a look if it is clearer now 🙏